src/styles/product-card.scss

```diff
@@ -0,0 +1,88 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: $cardWidth;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    object-fit: contain;
+    object-position: center;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    color: $textMenuColor;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondaryColor;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $textMenuColor;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondaryColor;
+  }
+
+  &__price-amount {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $textMenuColor;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    width: 166px;
+    background-color: $mainColor;
+    border-radius: 5px;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    text-align: center;
+    border: 1px solid $mainColor;
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 12th point. Fix this in all places in your code